src/game/sound_samples.c

```diff
@@ -84,6 +86,23 @@ int init_buffer_and_device(Sound_samples *sound_samples,
         }
     }
     
+    /* Allocating active audio buffer location*/
+    //TODO: Allocate one huge active audio buffer with length of the maximum of all audio buffer, instead of one active buffer for each audio
+    sound_samples->active_audio_buf_array = PUSH_LT(sound_samples->lt, nth_calloc(sound_samples->samples_count, sizeof(uint8_t*)), free);
+    if (sound_samples->active_audio_buf_array == NULL) {
+      log_fail("Failed to allocate memory for active audio buffer pointer array\n");
+      return -1;
+    }
+    for (size_t i = 0; i < sound_samples->samples_count; ++i) {
+      sound_samples->active_audio_buf_array[i] = PUSH_LT(sound_samples->lt, nth_calloc(sound_samples->audio_buf_size_array[i],
+                                                                                       sizeof(uint8_t)), free);
+      if (sound_samples->active_audio_buf_array == NULL) {
+        log_fail("Failed to allocate memory for active audio buffer array\n");
+        return -1;
```

Comment:
Use RETURN_LT here.

Reply:
Invalid because of https://github.com/tsoding/nothing/pull/1069/files#r325950050